lib/YaST/Kdump/StartUpPage.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles DnsServer Main Page.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::Kdump::StartUpPage;
+use strict;
+use warnings;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init();
+}
+
+sub init {
+    my $self = shift;
+    $self->{rbtn_shown} = $self->{app}->radiobutton({id => "\"EnableDisalbeKdump\""});
+    $self->{rbtn_enable} = $self->{app}->radiobutton({id => "\"enable_kdump\""});
+    $self->{rbtn_firmware} = $self->{app}->radiobutton({id => "\"enable_kdump\""});
+    $self->{btn_accept} = $self->{app}->button({id => 'next'});
```

Comment:
Same as inFirmwareAssistedDumpPage.pm: "next" button can be handled by NavigationBase.pm.

Reply:
Thanks, updated code.